### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by eneufeld**
Update how to debug

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,33 +137,35 @@ Publish packages with lerna to update versions properly across local packages, [
...
 #### Code Generation
 
-You cannot debug the code generation. In order to debug it, you need to execute the `org.eclipse.emfcloud.coffee.product.codegen` with the correct parameters.
+To debug the code generation you would need to attach to the process when it is started from the backend.
+Alternativly you can run the `codegen.product` or `codegen_cpp.product` with the correct parameters.
 
 #### Workflow Analyzer
 
-In order to debug, start the `org.eclipse.emfcloud.coffee.product.workflow.analyzer` product in debug mode. The root application is : `org.eclipse.emfcloud.coffee.workflow.analyzer.application.Application`.
+In order to debug, start the `workflowanalyzer.product` product in debug mode. The root application is : `org.eclipse.emfcloud.coffee.workflow.analyzer.application.Application`.
```
Comment (ndoschek) [inline README.md:149]:
```suggestion
In order to debug, start the `workflowanalyzer.product` product in debug mode. The root application is : `org.eclipse.emfcloud.coffee.workflow.analyzer.application.application`.
```

File: README.md [hunk 2]
```diff
@@ -137,33 +137,35 @@ Publish packages with lerna to update versions properly across local packages, [
...
-In order to debug, start the `org.eclipse.emfcloud.coffee.product.workflow.dsl` product in debug mode.
+In order to debug, start the `workflowdsl.product` product in debug mode.
 Please make sure to add `--WF_LSP=5017` parameter to the browser app. In the backend you need to pass `-startSocket` as an application argument.
 
 You can also use the predefined `RunSocketServer-Headless.launch` run config.
 
 #### Coffee Model Server
 
-Use the `org.eclipse.emfcloud.coffee.modelserver.CoffeeModelServerLauncher` class to start the Model Server.
+Use the `org.eclipse.emfcloud.coffee.modelserver.app.Application` Eclipse Application or the corresponding `modelserver.product` from `org.eclipse.emfcloud.coffee.product` to start the Model Server.
```
Comment (ndoschek) [inline README.md:164]:
```suggestion
Use the `org.eclipse.emfcloud.coffee.modelserver.app.application` Eclipse Application or the corresponding `modelserver.product` from `org.eclipse.emfcloud.coffee.product` to start the Model Server.
```

File: README.md [hunk 3]
```diff
@@ -137,33 +137,35 @@ Publish packages with lerna to update versions properly across local packages, [
...
 
 You can also simply use the predefined `WorkflowAnalyzerServer.launch` run config.
 
 #### Workflow DSL
 
-In order to debug, start the `org.eclipse.emfcloud.coffee.product.workflow.dsl` product in debug mode.
+In order to debug, start the `workflowdsl.product` product in debug mode.
 Please make sure to add `--WF_LSP=5017` parameter to the browser app. In the backend you need to pass `-startSocket` as an application argument.
 
 You can also use the predefined `RunSocketServer-Headless.launch` run config.
```
Comment (ndoschek) [inline README.md:160]: This predefined launch config does not work anymore.

**Review by ndoschek — COMMENTED** [review]
Thanks for the update Eugen!
I noticed some minor typos and it seems to me that the `RunSocketServer-Headless.launch` config is out of date, some plugins seem to be missing (at least on my side?).
In general, I am wondering if we should provide launch configs for our products (+ a group launch config for all available products) to simplify debugging?

File: README.md (hunk 3 above)
Comment (eneufeld) [reply README.md:160]: I suggest to merge the readme change and fix the launch config in a separate commit, @ndoschek what do you think?

**Review by ndoschek — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (ndoschek) [reply README.md:160]: See https://github.com/eclipse-emfcloud/coffee-editor/issues/380

**Pull request merged**